hosting_docs/source/reference/1-commcare-cloud/commands.md

```diff
@@ -30,7 +30,7 @@ commcare-cloud [--control] [--control-setup {yes,no}] <env> <command> ...
 
 server environment to run against
 
-## Options
+## Optional Arguments
```

Comment:
Do you know why this changed? It doesn't seem like that should happen in this PR.

Reply:
Not sure why I can't see my comment that I made a week ago, but I had resolved this issue. I think originally the issue was due to a difference in my local version of argparse.